src/main/java/nourl/mythicmetals/MythicMetals.java

```diff
@@ -47,6 +47,8 @@ public void onInitialize() {
 
         RegisterSounds.register();
         FieldRegistrationHandler.register(MythicItems.class, MOD_ID, false);
+        FieldRegistrationHandler.register(MythicItems.Ingots.class, MOD_ID, false);
```

Comment:
You might be able to get away by setting the `recurseIntoInnerClasses` to `true` here, saving a few lines of register calls

Reply:
Would break how I add things to the Given material Maps